transport/http/constants.go

```diff
@@ -21,6 +21,15 @@
 package http
 
 const (
+	// ApplicationHeaderPrefix is the prefix added to application headers over
+	// the wire.
+	ApplicationHeaderPrefix = "Rpc-Header-"
+
+	// ContextHeaderPrefix is the prefix added to context headers over the wire.
+	ContextHeaderPrefix = "Context-"
```

Comment:
Almost makes me wonder if this should default to Rpc-Context so everything defaults under Rpc-